api/authenticator.py

```diff
@@ -2817,6 +2808,6 @@ class BaseSAMLAuthenticationProvider(
 
     FLOW_TYPE = "http://librarysimplified.org/authtype/SAML-2.0"
 
-    SETTINGS = SAMLSettings()
+    SETTINGS = SAMLSettings()  # type: ignore
 
-    LIBRARY_SETTINGS = []
+    LIBRARY_SETTINGS: List[dict] = []
```

Comment:
Is the type of this `LIBRARY_SETTINGS` the same as the others below? There's one that has a stronger `List[Dict[str, Any]]` type.

Reply:
Yeah, I think it is, [updated](https://github.com/ThePalaceProject/circulation/pull/652/commits/559aabf09aa167941018c7967c51d92aebb422ca)